x/application/types/errors.go

```diff
@@ -3,10 +3,13 @@ package types
 // DONTCOVER
 
 import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	errorsmod "cosmossdk.io/errors"
 )
 
 // x/application module sentinel errors
 var (
-	ErrSample = sdkerrors.Register(ModuleName, 1100, "sample error")
+	ErrAppInvalidStake   = errorsmod.Register(ModuleName, 1, "invalid application stake")
+	ErrAppInvalidAddress = errorsmod.Register(ModuleName, 2, "invalid application address")
+	ErrAppUnauthorized   = errorsmod.Register(ModuleName, 3, "unauthorized application")
+	ErrAppStakeAmount    = errorsmod.Register(ModuleName, 4, "invalid application stake amount")
```

Comment:
Is there a particular reason for this error to exist independently of `ErrAppInvalidStake`? I was expecting the justification to be its usage in a test assertion but I didn't see one.